Library/Edge.cs

```diff
@@ -35,28 +39,34 @@ public class Edge
         public Edge prev2;
         public Loop loop; // first node of the list of faces that use this edge. Navigate list using radial_next
 
-        /**
-         * Tells whether a vertex is one of the extremities of this edge.
-         */
+        /// <summary>
+        /// Whether the specified <see cref="Vertex"/> is one of the vertices that make up the <see cref="Edge"/>.
+        /// </summary>
+        /// <param name="v">The <see cref="Vertex"/> to compare.</param>
+        /// <returns><c>true</c> if <paramref name="v"/> is used by the <see cref="Edge"/>; otherwise, <c>false</c>.</returns>
         public bool ContainsVertex(Vertex v)
         {
             return v == vert1 || v == vert2;
         }
 
-        /**
-         * If one gives a vertex of the edge to this function, it returns the
-         * other vertex of the edge. Otherwise, the behavior is undefined.
-         */
+        /// <summary>
+        /// Returns the other <see cref="Vertex"/> that makes up the <see cref="Edge"/> of the specified <see cref="Vertex"/>.
+        /// </summary>
+        /// <remarks>Assumes that the specified <see cref="Vertex"/> is one of the vertices that make up the <see cref="Edge"/>; otherwise, behavior is undefined.</remarks>
+        /// <param name="v">The <see cref="Vertex"/> to get the other of.</param>
+        /// <returns>The other <see cref="Vertex"/> that makes up the edge. Shorthand for <c><paramref name="v"/> == <see cref="vert1"/> ? <see cref="vert2"/> : <see cref="vert1"/></c>.</returns>
         public Vertex OtherVertex(Vertex v)
         {
             Debug.Assert(ContainsVertex(v));
             return v == vert1 ? vert2 : vert1;
         }
 
-        /**
-         * If one gives a vertex of the edge to this function, it returns the
-         * next edge in the linked list of edges that use this vertex.
-         */
+        /// <summary>
```

Comment:
Precision:

```
<remarks
It is ensured that calling <c>Next<c> repeatedly on the result edge will iterate through all edges whose <paramref name="v"/> is an extremity.
<example>For instance the following iterates through all edges:
<code>
Edge firstEdge = edge;
do {
    // do something with `edge`
    edge = edge.Next(v);
} while(edge != firstEdge);
</code>
</example>
<seealso cref="Vertex.NeighborEdges"/>
</remarks>
```

(btw idk if using "made up" is more appropriate than using "extremity" as you replaced above but typically in this sentence above I find it more convenient to use "extremity". If you have a better word though feel free to suggest)